pkg/agent/route/route_linux.go

```diff
@@ -642,6 +642,15 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet, localAntreaFl
 		}...)
 	}
 
+	if c.connectUplinkToBridge {
+		writeLine(iptablesData, []string{
```

Comment:
Could you add some comments to explain why this rule is needed?

Reply:
Added.